virt/virt-getting-started.adoc

```diff
@@ -70,3 +70,4 @@ VMs are connected to the pod network by default. You must configure a secondary
 * xref:../virt/virtual_machines/virt-automating-windows-sysprep.adoc#virt-automating-windows-sysprep[Automate Windows VM deployments with `sysprep`].
 * xref:../virt/live_migration/virt-live-migration.adoc#virt-live-migration[Live migrate VMs].
 * xref:../virt/backup_restore/virt-backup-restore-overview.adoc#virt-backup-restore-overview[Back up and restore VMs].
+* link:https://access.redhat.com/articles/6994974[Tune and scale your cluster]
```

Comment:
This link and the other lead to the same page, but the friendly reference text is different. Just wondering if you think it should be the same, or as is.

Reply:
I changed the link text on this one from the title of article to match the style of the other links in the resources section so it didn't stick out like a lil odd duck.